NEWS.md

```diff
@@ -1,8 +1,8 @@
-# SKELETON
+# hySpc.read.jdx 0.0.x 2021-xx-xx
 
 ## Acknowledgements
-The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during summer 2020.
-Erick and the team carried out a significant overhaul of `hyperSpec` which led to this release.
+The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during Summer 2020 and Sang Truong during summer 2021.
```

Comment:
Were Erick working on this pacakge? Should he be mentioned here? @bryanhanson

Reply:
No Erick on this one.  Good catch!

Reply:
I corrected it.